test/JSON-Schema-Test-Suite/tests/draft7/optional/format/uuid.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "description": "validation of uuids",
+        "schema": {
+            "format": "uuid"
+        },
+        "tests": [
+            {
+                "description": "a valid uuid",
+                "data": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+                "valid": true
+            },
+            {
+                "description": "an invalid uuid with shorter length",
+                "data": "3fa85f64-5717-4562-b3fc-2c963",
+                "valid": false
+            },
+            {
+                "description": "an invalid uuid with missing hyphens",
+                "data": "3fa85f6457174562b3fc2c963f66afa6",
+                "valid": true
+            },
+            {
+                "description": "an invalid uuid with hyphens at wrong position",
+                "data": "3fa85f645-717-4562b-3fc2c9-63f66afa6",
+                "valid": true
+            }
+        ]
+    }
+]
```

Comment:
Where did you get this file from? Did you write it yourself? If so, maybe you should also send it to the JSON-Schema-Test-Suite-project.

Is `uuid` an officially mentioned/supported format-definition by the standard?

Reply:
Yes it is officially supported, in a newer standard than this lib currently supports though: 
https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.5